tests/5.0/declare_target/test_declare_target_nested_functions.F90

```diff
@@ -0,0 +1,69 @@
+!===--- test_declare_target_nested_functions.F90 -------------------------------------===//
+!
+! OpenMP API Version 5.0 Nov 2018
+!
+! Test of declare target on a host function that calls another host
+! function. According to 5.0 specification, the inner function, while
+! not explicitly declared on the target, should be treated as if it was.
+! The declared functions increment the passed-in value and return it. 
+! They are tested in a simple target region and the result is mapped out
+! and checked.
+!
+!//===----------------------------------------------------------------------===//
+
+#include "ompvv.F90"
+
+#define N 1024
+
+PROGRAM test_declare_target_nested_functions
+
+    USE iso_fortran_env
+    USE ompvv_lib
+    USE omp_lib
+    implicit none
+    INTEGER :: errors
+    
+    !$omp declare target to(outer_fn)
+    
+    errors = 0 
+     
+    OMPVV_TEST_OFFLOADING
+
+    OMPVV_TEST_VERBOSE(test_declared_functions() .ne. 0)
+
+    OMPVV_REPORT_AND_RETURN()
+
+CONTAINS  
+    INTEGER FUNCTION inner_fn(a)
+      INTEGER:: a
+      
+      OMPVV_ERROR_IF(omp_is_initial_device(), "Target region executed on host&
+      & instead of device, test fails on the host")
+     
+       inner_fn = 1 + a
+    END FUNCTION inner_fn
+
+    INTEGER FUNCTION outer_fn(a)
+      INTEGER:: a
+      
+      OMPVV_ERROR_IF(omp_is_initial_device(), "Target region executed on host&
```

Comment:
The long message is causing xlf to throw an error. Temp solution would be to shorten the msg. 
We should look into the MACRO.